swagger.json

```diff
@@ -2052,6 +2052,35 @@
               "type": "string"
             }
           ],
+          "get": {
+              "tags": [
+                  "Default",
+                  "Fleet"
+              ],
+              "summary": "/fleet/vehicles/{vehicle_id | external_id }",
+              "description": "Get a vehicle given either a vehicle ID or external ID (alphanumeric string).",
```

Comment:
Rather than alphanumeric string, we may want to reference the external id example we've used throughout this doc. (it's really a key:value rather than an alphanumeric string)